.github/workflows/create_ensemble.yaml

```
@@ -31,15 +31,15 @@ jobs:
 
     - name: Install dependencies
       run: |
-        install.packages(c("hubEnsembles", "dplyr", "lubridate", "purrr", "yaml", "argparser", "remotes"))
+        install.packages(c("hubEnsembles", "dplyr", "lubridate", "purrr", "yaml", "argparser", "remotes", "RcppTOML"))
```

```suggestion
        install.packages(c("hubEnsembles", "dplyr", "lubridate", "purrr", "yaml", "argparser", "remotes"))
```

This is only required for `update-target-data.yaml` file here https://github.com/CDCgov/covid19-forecast-hub/blob/54627aeb384691e4eb1563eb7ebe687268b43ecc/.github/workflows/update-target-data.yaml#L33